SimPEG/EM/Static/SIP/Regularization.py

```diff
@@ -0,0 +1,215 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from SimPEG import Utils, Maps, Mesh
+import numpy as np
+import scipy.sparse as sp
+from SimPEG.Regularization import BaseRegularization, Simple
+
+
+class MultiRegularization(BaseRegularization):
```

Comment:
Do we need to add this back? or can this be replaced by the new objective function implementation?

Reply:
@sgkang: what are your thoughts on this?

Reply:
I'll deprecate this.